### PR TITLE
Add the new forwarding including committee user.

### PR DESCRIPTION
**Pull request by reiterl**

### CODE REVIEW

File: tests/system/action/motion/test_create_forwarded.py [hunk 1]
```diff
@@ -63,6 +64,7 @@ def test_correct_origin_id_set(self) -> None:
                 "all_origin_ids": [12],
             },
         )
+        assert model.get("forwarded")
```
Comment (r-peschke) [inline tests/system/action/motion/test_create_forwarded.py:67]: Would you please use the optional field reason to check it overwrites the motions reasons

**Review by r-peschke — CHANGES_REQUESTED** [review]
Just 2 small things so far.-)

File: tests/system/action/motion/test_create_forwarded.py (hunk 1 above)
Comment (reiterl) [reply tests/system/action/motion/test_create_forwarded.py:69]: Added a test for 'reason'.

**Review by r-peschke — APPROVED** [review]
Except the dummy user

**Review by jsangmeister — APPROVED** [review]
Looks good to me, but @r-peschke should probably double-check it, since he wrote the concept

File: tests/system/action/motion/test_create_forwarded.py [hunk 2]
```diff
@@ -50,19 +54,94 @@ def test_correct_origin_id_set(self) -> None:
                 "meeting_id": 2,
                 "origin_id": 12,
                 "text": "test",
+                "reason": "reason_jLvcgAMx",
             },
         )
         self.assert_status_code(response, 200)
+        model = self.assert_model_exists(
```
Comment (r-peschke) [inline tests/system/action/motion/test_create_forwarded.py:61]: Please test the correct submitter in target-motion

File: tests/system/action/motion/test_create_forwarded.py [hunk 3]
```diff
@@ -28,6 +30,7 @@ def setUp(self) -> None:
             "motion_state/34": {
                 "name": "name_state34",
                 "meeting_id": 2,
+                "allow_motion_forwarding": True,
```
Comment (r-peschke) [inline tests/system/action/motion/test_create_forwarded.py:33]: The test data isn't correct. motion_state/34 is a state of the target-meeting, but is also used in the source motion/12 of meeting/1 as current state.
All this data is valid only in their own meeting and cannot be used in the other meeting, too.
The target-motion should use the default workflow of the new meeting with the first state of it.
Please check, if there are more fields form source-motion used in target-motion, i.e. amendments, attachments. The may not be copied to the target motion.
Use tests to show the behavior in this cases.

**Review by r-peschke — CHANGES_REQUESTED** [review]

File: tests/system/action/motion/test_create_forwarded.py (hunk 2 above)
Comment (reiterl) [reply tests/system/action/motion/test_create_forwarded.py:61]: Add code to test for submitter, motion_submitter and link in user/2.

File: tests/system/action/motion/test_create_forwarded.py (hunk 3 above)
Comment (reiterl) [reply tests/system/action/motion/test_create_forwarded.py:37]: Added a state 30 of the source meeting.
Check the state_id of the created motion.
The create_forwarded action is a **create action**, so no coping of amendmens, etc. No need to test that.

**Review by r-peschke — APPROVED** [review]

**Pull request merged**